app/grants/templates/grants/index.html

```diff
@@ -146,6 +147,7 @@
           {% include 'grants/detail/side-cart.html' %}
         </div>
       </div>
+      {% endcomment %}
 
 
```

Comment:
There's a missing closing tag here (`</div>`) - That's what has knocked the footer out :)